jvm-annotations/src/main/java/org/quickperf/jvm/allocation/ByteAllocationMeasureFormatter.java

```diff
@@ -13,25 +13,42 @@
 
 package org.quickperf.jvm.allocation;
 
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+import java.text.NumberFormat;
+import java.util.Locale;
+
 public class ByteAllocationMeasureFormatter {
 
     public static final ByteAllocationMeasureFormatter INSTANCE = new ByteAllocationMeasureFormatter();
 
+    private final static DecimalFormat BYTE_PREFIX_FORMATTER = (DecimalFormat) NumberFormat.getInstance(Locale.ENGLISH);
```

Comment:
From https://docs.oracle.com/javase/7/docs/api/java/text/DecimalFormat.html : " It is recommended to create separate format instances for each thread. If multiple threads access a format concurrently, it must be synchronized externally."
Please create on instance for each call